contacts.html

```diff
@@ -0,0 +1,118 @@
+<!DOCTYPE html>
+<html lang="ru-RU">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Контакты</title>
+    <link rel="stylesheet" href="css/style.css">
+	<style>
```

Comment:
Why do you write styles global in HTML when you have a a linked style file?

Reply:
I won't do it again.